### PR TITLE
Feature/presenter test

### PR DESCRIPTION
**Pull request by stasfly**
Fulfill the lack of tests fo the app.
https://trello.com/c/rLJiy1SD/106-cover-the-app-with-tests

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_25_093845) do
+ActiveRecord::Schema.define(version: 2022_08_27_064529) do
```
Comment (mmrshk) [inline db/schema.rb:13]: why this was changed?

File: spec/presenters/article_presenter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+require 'rails_helper'
+
+RSpec.describe ArticlePresenter do
+  include ActionView::TestCase::Behavior
+
+  let(:tags_list)     { create_list(:tag, 2) }
+  let(:valid_article) { create(:article, :published, :uk, :with_image_url_inside_text, tags: tags_list) }
+
+  subject!(:article_presenter) { described_class.new(valid_article) }
```
Comment (mmrshk) [inline spec/presenters/article_presenter_spec.rb:11]: subject without bang.

Comment (mmrshk) [reply spec/presenters/article_presenter_spec.rb:11]: and move to the top

File: spec/presenters/article_presenter_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  describe '#article' do
+
```
Comment (mmrshk) [inline spec/presenters/article_presenter_spec.rb:14]: remove empty line

File: spec/presenters/article_presenter_spec.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  let(:tags_list)     { create_list(:tag, 2) }
+  let(:valid_article) { create(:article, :published, :uk, :with_image_url_inside_text, tags: tags_list) }
+
+  subject!(:article_presenter) { described_class.new(valid_article) }
+
+  describe '#article' do
+    
+    it 'returns correct values' do
+      expect(subject.article).to be valid_article
```
Comment (mmrshk) [inline spec/presenters/article_presenter_spec.rb:16]: please, use subject everywhere, not article_presenter

File: spec/presenters/article_presenter_spec.rb (hunk 2 above)
Comment (mmrshk) [reply spec/presenters/article_presenter_spec.rb:11]: `subject { described_class.new(valid_article) }`

File: spec/presenters/article_presenter_spec.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      expect(article_presenter).to be_an ArticlePresenter
+      expect(article_presenter.all_tags).to eq(valid_article.tags.pluck(:name).join(', '))
```
Comment (mmrshk) [inline spec/presenters/article_presenter_spec.rb:18]: check each method in different its

**Review by mmrshk — CHANGES_REQUESTED** [review]
Refactor

File: db/schema.rb (hunk 1 above)
Comment (stasfly) [reply db/schema.rb:13]: I did a migration after pool main branch. I suppose it has made this change

**Review by mmrshk — APPROVED** [review]
lgtm

**Pull request merged**